src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+        'Icons and tags.',
+        'Icon buttons at the end of the table row for touch screens.',
+        `Checkboxes to select rows.`,
+        `Status stripes for row items with special statuses.`,
+    ]}
+    gridImageProps={{ sm: 6, md: 4 }}
+/>
+
+You should seek to make your tables both easy to navigate and effective to use.
+
+<ImageGrid
+    images={[TableCellUnderlinedField, TableCellImage]}
+    captionsUnderImages={[
+        'CAUTION: Be careful when using underlined text fields in a table, as their understrokes compete with the line strokes between each table row.',
+        `AVOID: Images can heighten table rows. Consider using a list instead.`,
+    ]}
+/>
+
+# Behaviors
+
+## Row Actions
+
+Similar to lists, tables also have row actions. Common row actions include edit, delete and view details. They are usually placed at the end of a table row.
+
+<ImageGrid
+    images={[ActionAtRowEnd, ActionOnHover]}
+    captionsUnderImages={[
+        `Row actions can be placed at the end of the row.`,
+        `DO: On desktop, you can choose to have row actions visible only when the mouse is hovering on the row. The buttons won't compete for rooms with the rest of the table.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[Drag1, Drag2]}
+    caption={[
+        `Drag-and-drop actions are usually placed at the beginning of a row. If you have row numbers, you may choose to display the drag handle on hover.`,
+    ]}
+/>
+
+## Batch Actions
+
+If your application allows users to perform batch actions to their data set, such as deleting 15 items at once, you can add a checkbox column on the left rim of the table.
+
+<ImageGrid
+    images={[BatchAction1, BatchAction2]}
+    captionsUnderImages={[
+        `Buttons are disabled when no rows are checked.`,
+        `Buttons become enabled when rows are checked. Optionally, use theme color to highlight the selected row, and label the button with the number of items selected to account for pagination.`,
+    ]}
+/>
+
+<!--
+
+Note: I think this is a good suggestion, but might be too high of a requirement for our adopters
+
+You may change the clickable area when users are cognitively loaded and busy deciding which row to check. When they enter this "batch action mode", increase the row selection area so that it is easier to select.
+
+<ImageGrid
+    images={[ClickArea1, ClickArea2]}
+    captionsUnderImages={[
+        `When nothing is selected, the selection target area is limited to the region around the checkbox.`,
+        `When a row is selected, the application enters this "batch action mode", and users can select other rows by clicking anywhere on those rows.`,
+    ]}
+/>
+-->
+
+## Sticky Header Row
+
+When a table gets long, your users should not need to scroll back to the top to view the table header.
+
+<ImageGrid
+    images={[StickyHeader]}
+    caption={[`Use sticky headers to help your users navigate through the table.`]}
+    regularWidth
+/>
+
+## Expanded Row
```

Comment:
This one worries me a bit from an implementation standpoint. It feels less like a table with this kind of behavior. How would this affect sorting and filtering?

Reply:
The expanded part would be viewed as "attached" to the original content, and would not affect the sorting / filtering at all.